tests/mapped_struct.py

```diff
@@ -1731,7 +1771,7 @@ def testMixed(self):
         self.assertTrue(big >= small)
         self.assertFalse(big > big)
 
-        big = self.pack([1L, 2L, 3L])
+        big = self.pack([1, 2, 3])
```

Comment:
They need to be longs. Use `long(1)` maybe as an easy way to make it dual python?